modules/oe_theme_helper/src/Plugin/Block/PageHeaderBlock.php

```diff
@@ -99,4 +123,34 @@ public function setTitle($title): self {
     return $this;
   }
 
+  /**
+   * Constructs a new PageHeaderBlock instance.
+   *
+   * @param array $build
+   *   Header build array.
+   *
+   * @return array
+   *   Header build array with the a breadcrumb (if any is applicable)
```

Comment:
The phpdoc here is wrong. You can also simplify the param and return docs with `array $build A render array` and `@return array the processed render array` or something like that.